docs/Allocene/style/main-games.css

```diff
@@ -0,0 +1,688 @@
+@import url("./header.css");
+@import url("./footer.css");
+@import url("./font.css");
+
+.search {
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 60px;
+  width: 100%;
+  margin-top: 70px;
+  background-color: #221F1F;
+  color: #FFF;
+}
+
+.search__contain {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 90%;
+}
+
+.search__form {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+select[name="genre"] {
+  border-radius: 6px;
+  color: #FFF;
+  background-color: #333;
+  width: 114px;
+  height: 36px;
+  border: 0;
+  outline: 0;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: 0;
+  text-align: left;
+  padding-left: 12px;
+}
+
+select option {
```

Comment:
add class

Reply:
Done

Reply:
Cool, now you have one class selector instead of cascade of two tag selectors